semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/SymbolOps.scala

```diff
@@ -144,6 +148,60 @@ trait SymbolOps { self: Scalacp =>
   }
 
   implicit class XtensionSymbol(sym: Symbol) {
+    def isJavaDefined: Boolean = scalacp.isJavaDefined.get(sym) match {
+      case Some(cachedResult) =>
+        cachedResult
+      case _ =>
+        val result =
+          if (sym.isReallyPackage) false
+          else if (sym.isPackageObject) false
```

Comment:
`sym.isPackageObject` is defined as `def isPackageObject: Boolean = sym.name == "package"`. Is there something in javac that prevents programmers from calling their classes `package`?

Reply:
Good catch. I removed this guard, it was there from an early experiment that was obsolete once we gained the ability to query the classpath index.